### DISCUSSION TITLE
I am encountering an error when attempting to install a module I created from GitHub.

### DISCUSSION THREAD

**Discussion by heckmon** [micropython/micropython#13744] [Libraries & Drivers] [answered]
I am new here. So please correct me if I am wrong

I created a library and uploaded it to a GitHub repo. But this is what I get when I try to install it from mip.(my board is ESP32-WROOM-32)

This is what I followed:
https://github.com/jimmo/micropython-mlx90640
by @jimmo

@jimmo help

Here is the error:

```
mip.install("github:AthulNoobie/micropython_7segment_display_library")
Installing github:AthulNoobie/micropython_7segment_display_library/package.json to /lib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mip/__init__.py", line 1, in install
  File "mip/__init__.py", line 1, in _install_package
  File "mip/__init__.py", line 1, in _install_json
  File "urequests.py", line 180, in get
  File "urequests.py", line 76, in request
OSError: -202
```

Here are the files in my GitHub repo:

images
LICENSE
README.md
__init\__.py
micropython_7seg.py
package.json

Here are the contents in package.json:

```json
{
  "urls": [
    ["__init__.py", "github:AthulNoobie/micropython_7segment_display_library/__init__.py"],
    ["micropython_7seg.py", "github:AthulNoobie/micropython_7segment_display_library/micropython_7seg.py"]
  ],
  "deps": [
    ["machine", "time"]
  ],
  "version": "1.0.0"
}
```

**Comment by mcauser** [comment, marked as answer]
In your package.json, the repo name doesn't match your repo on github.

https://github.com/AthulNoobie/micropython_7segmet_display_library

```diff
- 7segmet
+ 7segment
```

I'd suggest going into settings on github and renaming your repo to match, then it should work.
